app/src/main/res/layout/profile_progress_recently_played_story_card.xml

```diff
@@ -35,6 +35,7 @@
         android:layout_height="0dp"
         android:contentDescription="@{viewModel.computeLessonThumbnailContentDescription()}"
         android:importantForAccessibility="no"
+        android:scaleType="fitCenter"
```

Comment:
why we have to add a scaleType here

Reply:
because it was added in sw600dp0-land so just for being sure its absence doesnt create a problem

Reply:
but it was for tablet and now it will also be applied to mobile

Reply:
We don't need `android:scaleType="fitCenter"` in `LessonThumbnailImageView `because it has already been implemented in kotlin files. https://github.com/oppia/oppia-android/blob/3a0afb4eb8222d4ff3acd2e2e660b85911d35229/app/src/main/java/org/oppia/android/app/customview/LessonThumbnailImageView.kt#L110

Reply:
ok i will delete it